distribution/lib/Standard/Base/0.0.0-dev/src/Data/Noise/Generator.enso

```diff
@@ -29,8 +29,8 @@ type Generator
 
        The return type may be chosen freely by the generator implementation, as
        it usually depends on the generator and its intended use.
-    step : Number -> Interval -> Any
-    step _ _ = unimplemented "Only intended to demonstrate an interface."
+    step : Generator -> Number -> Interval -> Any
```

Comment:
Given we haven't done this anywhere else probably not worth this change.

Reply:
Thanks, yes